doc/quickstart/cpp.md

```diff
@@ -20,6 +20,13 @@ command
 sudo apt-get install libmlpack-dev
 ```
 
+**Note for Ubuntu LTS Users**: The libmlpack-dev version in the Ubuntu LTS 
+repositories may not always be the latest. This can lead to issues, such as
+missing header files (e.g., mlpack.hpp missing in versions prior to 4.0). To 
+ensure compatibility with the latest mlpack features and examples, we recommend
+ either building mlpack from source or using Docker, as explained in 
```

Comment:
Suggest to omit mention of Docker, as that can muddle the problem further.

If a novice user is unable to figure that there is a clash between old and new versions of mlpack, I don't think sending them to the Docker path is helpful.

An advanced user will be able to figure out that Docker is an option anyway, by reading the instructions.

Reply:
Sure, that sounds like a good idea. I'm even thinking that all the second part of the paragraph might be unnecessary. But I guess it doesn't hurt to have it.

Anyway, I committed a version with the docker part removed. Thanks for the feedback!